src/reporter/app.py

```diff
@@ -21,3 +27,16 @@ async def report_endpoint(
     await send_observation(
         project_name=project_name, observation=observation, http_client=http_client
     )
+
+
+
+@app.post("/report/email")
+async def report_email_endpoint(payload: ReportPayload, graph_client: Annotated[GraphServiceClient, Depends(build_graph_client)]):
+    content=build_report_email_content(name=payload.name, version=payload.version, inspector_url=payload.inspector_url, rules_matched=payload.rules_matched, additional_information=payload.additional_information)
```

Comment:
There's no way this line is less than or equal to 120 columns. Maybe ruff isn't working?

https://github.com/vipyrsec/dragonfly-reporter/blob/main/pyproject.toml#L42

Reply:
Hm, you're right. I knew that line was over 120 (was about 200) when I was writing it, but I figured Ruff would fix that and didn't give it a second thought. Maybe it's too long for Ruff? Either way, I can fix it manually